client/src/components/ListItems.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+
+interface ListItemsProps {
+  value: string
+  forceList?: boolean
+}
+
+const ListItems = ({ value, forceList }: ListItemsProps) => {
+  const items = value.split(/\r\n|\r|\n|\v|\f|\u2028|\u2029/)
```

Comment:
This can lead to empty list items; maybe split on "one or more" consecutive newlines?